src/Propel/Generator/Builder/Util/SchemaReader.php

```diff
@@ -460,7 +463,9 @@ private function getLocationDescription()
             $location .= sprintf('file %s,', $this->currentXmlFile);
         }
 
+        /** @psalm-suppress InvalidArgument */
```

Comment:
We usually have baselines for this, I am not sure we should start adding ignores and suppress then directly into code.

Reply:
This may not be required anymore, but there is a typing conflict between 7.4 and 8.0.  These functions require a `resource` and not an instance of `\XMLParser`.  I believe, now that `$parser` is annotated as mixed it's probably okay to remove.